zipline/finance/blotter/simulatedblotter.py

```diff
@@ -19,6 +19,7 @@
 from six import iteritems
```

Comment:
Can you rename this to `simulated_blotter.py`?

Reply:
Done